components/time-picker/index.tsx

```diff
@@ -51,6 +57,9 @@ const TimePicker = React.forwardRef<any, TimePickerProps>(
 );
 
 TimePicker.displayName = 'TimePicker';
+TimePicker.defaultProps = {
```

Comment:
The default value does not seem to be necessary.

Reply:
Done